### ISSUE TITLE
Is there any way to see all the bookmarks and name them?

### ISSUE THREAD

**Issue by Jaurocha** [Konamiman/SuperBookmarks#3]
Is there any way to see all the bookmarks and name them?

In the built in bookmark you have the ability to see them all as a sort of index making it possible to build a logical view that you navigate. 

Any plans on making something similar?

**Comment by georgethejournalist** [comment]
Seconded, this would be very useful!

**Comment by ScottHutchinson** [comment]
Yeah, without this feature, I can't really make use of this extension. The main problem I need solved is for Visual Studio to save my bookmarks, breakpoints, and other options(?) immediately, not when exiting VS (by which time, VS has crashed). I'm still looking for a solution to this problem, and I'm amazed that VS still works this way after all these years.

**Comment by billw2012** [comment]
This is a critical feature, not sure how much use this really is without it.

**Comment by ShiverZm** [comment]
i need this feature for this plugin,without this feature ,this plugin does not matter for me.
